### ISSUE TITLE
some hyperparameter detail problem in the code

### ISSUE THREAD

**Issue by pyjnqd** [chmxu/eTT_TMLR2022#2]
hello, it's a fantastic work which achieve a ViT-applied adapter. It can expand adapter method from Resnet to ViT-like network. Could you provide some suggestions on how to choose a good learning rate with adam/adamw? Also, I wanna know how the hyperparameters in code you get. Appreciate your efforts on it.

**Comment by chmxu** [comment]
Hi, the learning rate settings are given in my code and are tuned via validation. Can you provide your reproduced results so that I can check if there are any problems in the current version?

**Comment by pyjnqd** [comment]
Sorry, the abnormal results is invalid because of my fault when I run the codes, Now it performs better and reasonably. Thanks for your reply. Happy weekend.
